apps/client/src/app/components/admin-market-data/create-asset-profile-dialog/create-asset-profile-dialog.component.ts

```diff
@@ -117,7 +117,11 @@ export class GfCreateAssetProfileDialogComponent implements OnInit, OnDestroy {
         })
         .pipe(takeUntil(this.unsubscribeSubject))
         .subscribe(() => {
-          this.dialogRef.close();
+          this.dialogRef.close({
+            dataSource: 'MANUAL',
```

Comment:
`dataSource` won't work like this. You need to extend `dataProviders` of `AdminData` by `useForExchangeRates` (`DataProviderService` → `getDataSourceForExchangeRates()`):

```
dataProviders: (DataProviderInfo & {
  assetProfileCount: number;
  useForExchangeRates: boolean; <--
})[];
```